html_css/Exercise_1/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Exercise_1</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body class="body_attributes">
+
+<div class="white_box">
+	<img class="logo" src="img/ex1_brand_icon.png">
+	<br>
+	<label for="uname"><img class="user_img" src="img/ex1_username_icon.png"></label>
```

Comment:
user complete variable names (eg `username` instead of `uname`) for other developer's readability